packages/react-native-renderer/src/ReactFabricHostConfig.js

```diff
@@ -68,9 +53,15 @@ type Node = Object;
 export type Type = string;
 export type Props = Object;
 export type Instance = {
+  // Reference to the shadow node.
   node: Node,
-  canonical: ReactFabricHostComponent,
-  ...
+  nativeTag: number,
+  viewConfig: ViewConfig,
+  currentProps: Props,
+  // Reference to the React handle (the fiber)
+  internalInstanceHandle: Object,
+  // Exposed through refs.
+  publicInstance: ReactFabricHostComponent,
```

Comment:
Could we structure this as the following instead, to create a single shared object between instances?

```
  // Reference to the shadow node.
  node: Node,
  canonical: {
    nativeTag: number,
    viewConfig: ViewConfig,
    currentProps: Props,
    // Exposed through refs.
    publicInstance: ReactFabricHostComponent,
},
```

Reply:
I was just implementing that when I see that comment. Making sure I'm updating all the references using this PR as well, hehe. Thanks!